src/test/java/net/sf/jabref/importer/fetcher/CrossRefTest.java

```diff
@@ -34,14 +35,30 @@ public void findTitleOnly() {
     }
 
     @Test
-    public void findIncompleteTitle() {
+    public void notFindIncompleteTitle() {
```

Comment:
As we base a lot of things on the DOI, e.g., full text retrieval we decided that we need a high accuracy here.